app/views/layouts/application.html.erb

```diff
@@ -11,18 +11,17 @@
   </head>
 
   <body>
-    <%= render 'layouts/navbar' %>
-    <%= yield %>
-    
-    <p class="notice"> <%= notice %> </p>
-    <p class="alert"> <%= alert %> </p>
-    
+    <% if !notice && !alert %>
+      <%= render 'layouts/navbar' %>
+      <%= yield %>
+      <%= render 'layouts/footer' %>
+    <% else %>
+      <%= render 'layouts/navbar' %>
+      <%= yield %>
+      <%= render 'layouts/footer' %>
+      <p class="notice"> <%= notice %> </p>
+      <p class="alert"> <%= alert %> </p>
+    <% end %>
```

Comment:
this logic can be splitted into a partial as _flash_message

Reply:
We are going to add a task to improve flashes in another task!
In this case, just implement that part so the styles don't fail me.